sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -5,7 +5,9 @@ module GenericWork
 
     include Sufia::Works::Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
-    include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Batches
```

Comment:
Not sure why these changes are needed for work.  Is this just a quick fix for the collection size issue https://github.com/projecthydra/sufia/issues/1120?  If so my preferred method would be to change https://github.com/projecthydra/sufia/blob/master/sufia-models/app/models/concerns/sufia/collection_behavior.rb#L21 to just return zero.